curvefs/src/mds/topology/topology_metric.cpp

```diff
@@ -61,6 +62,11 @@ void TopologyMetricService::UpdateTopologyMetrics() {
         }
     }
 
+    // clear fs inodeNum metric
+    for (auto &it : gFsMetrics) {
+        it.second->inodeNum_.set_value(0);
+    }
+
```

Comment:
it seems it's no sense.

Reply:
> it seems it's no sense.

If not set to zero, it will become cumulative